botorch/optim/optimize.py

```diff
@@ -1247,6 +1372,16 @@ def optimize_acqf_discrete_local_search(
         - a `q x d`-dim tensor of generated candidates.
         - an associated acquisition value.
     """
+    if batch_initial_conditions is not None:
+        if not (
+            len(batch_initial_conditions.shape) == 3
+            and batch_initial_conditions.shape[-2] == 1
+        ):
+            raise ValueError(
+                "batch_initial_conditions must have shape `n x 1 x d` if "
+                f"given (recieved {batch_initial_conditions})."
```

Comment:
This may not be very useful / possible to parse if `batch_initial_conditions` is large
```suggestion
                f"given (received shape {batch_initial_conditions.shape})."
```

Reply:
this was an oversight I meant it to be the shape